test/e2e/quick_start_test.go

```diff
@@ -197,9 +198,9 @@ func TestQuickStart(t *testing.T) {
 
 	t.Logf("Waiting for guest kubeconfig to become available")
 	var guestKubeConfigSecret corev1.Secret
-	err = wait.Poll(1*time.Second, 5*time.Minute, func() (done bool, err error) {
+	err = wait.PollUntil(1*time.Second, func() (done bool, err error) {
```

Comment:
should this not keep its own timeout apart from the overall timeout?

Reply:
Fixed